lib/cogctl/actions/chat_handles/create.ex

```diff
@@ -3,9 +3,9 @@ defmodule Cogctl.Actions.ChatHandles.Create do
   alias Cogctl.Table
 
   def option_spec do
-    [{:user, ?u, 'user', :string, 'Username of user to add handle to', :required},
```

Comment:
How come some options we're calling "required" used to have `:required`, but others didn't?

Also, if the `:required` atom isn't needed anymore, how is an option formally marked as required?